Curve-finance-suite/Unkill/src/agent.spec.ts

```diff
@@ -0,0 +1,90 @@
+import {
+  FindingType,
+  FindingSeverity,
+  Finding,
+  HandleTransaction,
+} from "forta-agent";
+import { createAddress, TestTransactionEvent } from "forta-agent-tools";
+import { provideHandleTransaction, createFinding } from "./agent";
+import { stablePoolInterface } from "./abi";
+
+const POOL_OWNER = createAddress("0x1");
+
+describe("Unkill Agent Test Suite", () => {
+  let handleTransaction: HandleTransaction;
+
+  beforeAll(() => {
+    handleTransaction = provideHandleTransaction(POOL_OWNER);
```

Comment:
Is is necessary for `handleTransaction` to be reset after every test?
From what I can see, there is no way that it would change between each test or affect the outcome of each test in any way.

Reply:
That code is executed only once before running the tests suite. It is a `beforeAll`, maybe you are thinking in a `beforeEach`.